### PR TITLE
Implement continious deployment [Oleksii]

### PR DESCRIPTION
**Pull request by vglfr**

### CODE REVIEW

File: .semaphore/backend-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          - echo "machine git.heroku.com
+                    login ${HEROKU_USER}
+                    password ${HEROKU_PASSWORD}" >> $HOME/.netrc
+          - git config --global user.email "${GIT_EMAIL}"
+          - git config --global user.name "${GIT_USER}"
+      env_vars:
+        - name: GIT_EMAIL
+          value: vf.velt@gmail.com
+        - name: GIT_USER
+          value: vglfr
```
Comment (anna-kuzii) [inline .semaphore/backend-deploy.yml]: is it correct to hard code your own email?

File: client/src/components/ProfilePasswordChange/index.tsx [hunk 2]
```diff
@@ -4,14 +4,10 @@ import { renderField } from '../CommonComponents/ReduxFormFields';
 
 import React from 'react';
 import { connect } from 'react-redux';
-import { Form, Row, Label, Button, Spinner, FormProps } from 'reactstrap';
+import { Form, Row, Label, Button, FormProps } from 'reactstrap';
 import { Field, reduxForm } from 'redux-form';
 import { Link } from 'react-router-dom';
 
-interface IStateToProps {
-    changeProfilePassword: any; // FIXME
```
Comment (anna-kuzii) [inline client/src/components/ProfilePasswordChange/index.tsx]: lol
I believe we requested  to change it to TODO

Comment (vglfr) [reply client/src/components/ProfilePasswordChange/index.tsx]: That's the consequence of recently set `no-unused-variables` tslint rule. This particular interface is not used yet.

File: .semaphore/backend-deploy.yml (hunk 1 above)
Comment (vglfr) [reply .semaphore/backend-deploy.yml]: One could find it in `git log` of the very commit. The principle you're referring to seem sound to me though; hence changes.

**Pull request merged**